python/pyspark/sql/tests/test_arrow.py

```diff
@@ -160,32 +160,6 @@ def test_toPandas_arrow_toggle(self):
         assert_frame_equal(expected, pdf)
         assert_frame_equal(expected, pdf_arrow)
 
-    def test_toPandas_respect_session_timezone(self):
-        df = self.spark.createDataFrame(self.data, schema=self.schema)
-
-        timezone = "America/New_York"
-        with self.sql_conf({
```

Comment:
@MaxGekk, I think we should only remove this test case.

Reply:
This is what I originally did but I had to remove the rest of the tests because the parts refer to vars from removed code. See:
https://github.com/apache/spark/pull/27169#issuecomment-573364949
https://github.com/apache/spark/pull/27169#issuecomment-573407522
https://github.com/apache/spark/pull/27169#issuecomment-573446744

Reply:
Can you just revive those configurations? Looks there are some more changes to fix together in the codes, and I think we should better do it separately.

Reply:
How about to extract removing of 'spark.sql.execution.pandas.respectSessionTimeZone'  to a separate PR, and remove other 2 configs here in this PR?

Reply:
Yeah, sorry for not being clear. I wanted to mean that.